AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -199,7 +248,8 @@ public final class PayToComponent: PaymentComponent,
             scopeInstance: self,
             postfix: ViewIdentifier.identifierPickerItem
         )
-        return item
+        // we return a container item to prevent the picker from becoming firstResponder
```

Comment:
Shouldn't this be decided in a view controller? This is clearly UIKit logic leaking to a business logic layer. I tried to check `FormContainerItemView` if it overrides `canBecomeFirstResponder` and I didn't find anything, can you guide me so I get the idea?

Reply:
Ideally yes. But if we don't add this the picker becomes first responder since we make the first eligible item first responder in FormViewController.
Or we try to add a property to this picker element to not trigger first responder.
Or we don't add it at all and have the picker auto open at start.